<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KukaRobotControlControls</class>
 <widget class="QWidget" name="KukaRobotControlControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>601</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Basics</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="5" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_33">
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Axis 2 (xyz):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_axis2_x_roboRegis">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_axis2_y_roboRegis">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_axis2_z_roboRegis">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_collectAxis2Points_roboRegis">
           <property name="text">
            <string>Collect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Point num: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_axis2PointNum_roboRegis">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_removeAxis2Previous_roboRegis">
           <property name="text">
            <string>Remove previous</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_selfCheck">
           <property name="text">
            <string>Self check movement</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Movemnt mode</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_moveMode">
           <item>
            <property name="text">
             <string>Move flange under roboBase</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Move flange under flange</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_32">
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Axis 1 (xyz):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_axis1_x_roboRegis">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_axis1_y_roboRegis">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_axis1_z_roboRegis">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_collectAxis1Points_roboRegis">
           <property name="text">
            <string>Collect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Point num: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_axis1PointNum_roboRegis">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_removeAxis1Previous_roboRegis">
           <property name="text">
            <string>Remove previous</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="3" column="5">
          <widget class="QPushButton" name="pushButton_ym">
           <property name="text">
            <string>Y-</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="4">
          <widget class="QPushButton" name="pushButton_xp">
           <property name="text">
            <string>X+</string>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <widget class="QPushButton" name="pushButton_yp">
           <property name="text">
            <string>Y+</string>
           </property>
          </widget>
         </item>
         <item row="1" column="6">
          <widget class="QPushButton" name="pushButton_rxp">
           <property name="text">
            <string>RX+</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QPushButton" name="pushButton_xm">
           <property name="text">
            <string>X-</string>
           </property>
          </widget>
         </item>
         <item row="3" column="7">
          <widget class="QPushButton" name="pushButton_rym">
           <property name="text">
            <string>RY-</string>
           </property>
          </widget>
         </item>
         <item row="4" column="4">
          <widget class="QPushButton" name="pushButton_zp">
           <property name="text">
            <string>Z+</string>
           </property>
          </widget>
         </item>
         <item row="8" column="5">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="6">
          <widget class="QPushButton" name="pushButton_ryp">
           <property name="text">
            <string>RY+</string>
           </property>
          </widget>
         </item>
         <item row="3" column="8">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="5">
          <widget class="QPushButton" name="pushButton_zm">
           <property name="text">
            <string>Z-</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>(mm or degrees)</string>
           </property>
          </widget>
         </item>
         <item row="4" column="7">
          <widget class="QPushButton" name="pushButton_rzm">
           <property name="text">
            <string>RZ-</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLineEdit" name="lineEdit_intuitiveValue">
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_6">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Step size =</string>
           </property>
          </widget>
         </item>
         <item row="4" column="6">
          <widget class="QPushButton" name="pushButton_rzp">
           <property name="text">
            <string>RZ+</string>
           </property>
          </widget>
         </item>
         <item row="1" column="7">
          <widget class="QPushButton" name="pushButton_rxm">
           <property name="text">
            <string>RX-</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton_recordInitial">
           <property name="text">
            <string>Collect position 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_goToInitial">
           <property name="text">
            <string>Go to position 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_record2">
           <property name="text">
            <string>Collect position 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_goto2">
           <property name="text">
            <string>Go to position 2</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Tool storage: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_toolStoragePath">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_loadToolStorage">
           <property name="text">
            <string>Load tool storage</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_connectKuka">
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_startTracking">
           <property name="text">
            <string>Start tracking</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="pushButton_printTcpUnderBase">
         <property name="text">
          <string>Print flange under internal base</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
